packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -337,7 +337,7 @@ For example, a Transaction object such as:
 }
 ```
 
-indicates that a `WEB` request with `id` `Wn4f-0RQ8D8lTSLkAmkKdQAADqo` created this event.
+indicates that a `WEB` request with `id` `Wn4f-0RQ8D8lTSLkAmkKdQAADqo` has created this event.
 
 | Property   | Description                                                                                             | DataType             | Nullable |
 | ---------- | ------------------------------------------------------------------------------------------------------- | -------------------- | -------- |
```

Comment:
Remove periods in table descriptions

Reply:
This seems to be a non-typical scenario, where a proper sentence is broken up with code in the middle. This probably needs discussion as a team.